recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -2,26 +2,25 @@ SUMMARY = "Web Platform for Embedded Framework"
 HOMEPAGE = "https://github.com/WebPlatformForEmbedded"
 SECTION = "wpe"
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1fe8768cbb5fd322f7d50656133549de"
 PR = "r0"
 
 require include/wpeframework.inc
+require include/compositor.inc
 
-# FIXME the compositor shares flags across wpeframework and wpeframework-plugins. Not sure if this is a good idea...
-include include/compositor.inc
+DEPENDS = "zlib python-jsonref-native virtual/egl  \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', '${WPE_COMPOSITOR_DEP}', '', d)} \
+"
 
-DEPENDS = "zlib python-jsonref-native virtual/egl ${WPE_COMPOSITOR_DEP}"
 DEPENDS_append_libc-musl = " libexecinfo"
 
 PV = "3.0+git${SRCPV}"
 
 SRC_URI = "git://github.com/WebPlatformForEmbedded/WPEFramework.git;protocol=git;branch=master \
            file://wpeframework-init \
            file://wpeframework.service.in \
-           file://0001-Thread.cpp-Include-limits.h-for-PTHREAD_STACK_MIN-de.patch \
```

Comment:
Why are you removing this? This breaks musl builds.

Reply:
In this version its part of the code:
https://github.com/WebPlatformForEmbedded/Thunder/blob/aed13867c93fe76dc29d32abe4237559cc19b62c/Source/core/Thread.cpp#L5